telluric/georaster.py

```diff
@@ -1082,8 +1090,13 @@ def copy_with(self, **kwargs):
         # unless totally necessary
         if 'image' not in init_args:
             init_args['image'] = self.image
+        _cls = self.__class__
+        if mutable:
+            _cls = MutableGeoRaster
+        return _cls(**init_args)
 
-        return self.__class__(**init_args)
+    def as_mutable(self):
+        return self.copy_with(mutable=True)
```

Comment:
why this is not enough `self.copy_with(mutable=True)`?

Reply:
Not a big deal, I think that is nicer and more understoodable